tests/python_tests/test_llm_pipeline_static.py

```diff
@@ -308,38 +381,57 @@ def end(self):
     tokenizer = Tokenizer(model_path)
     tokenized_input = tokenizer.encode(prompt)
 
-    pipe = LLMPipeline(model_path, "NPU", **config)
-    encoded_results = pipe.generate(tokenized_input, max_new_tokens=1000, ignore_eos=True, streamer=TestStreamer())
+    encoded_results = npu_model.generate(
+        tokenized_input, 
+        max_new_tokens=1000, 
+        ignore_eos=True, 
+        streamer=TestStreamer(),
+    )
 
     assert len(encoded_results.tokens[0]) == num_iters
 
 
 # FIXME: Known problem, output differs from stateful pipeline starting from 3rd prompt!
-@pytest.mark.skip(reason="JIRA-144780: Output differs from stateful pipeline")
-@pytest.mark.parametrize("config", pipeline_configs)
-@pytest.mark.parametrize("model_id", get_models_list())
 @pytest.mark.precommit
-def test_chat_generation(config, model_id):
+@pytest.mark.parametrize("llm_model", MODELS_LIST, indirect=True)
+@pytest.mark.parametrize("npu_config", PIPELINE_CONFIGS, indirect=True)
+def test_chat_generation(
+    ov_model: LLMPipeline,
+    npu_model: LLMPipeline,
+):
+    def generate_with_chat_mode(pipe: LLMPipeline, questions):
+        pipe.start_chat()
+        answers = [ pipe.generate(question, max_new_tokens=50, do_sample=False) for question in questions ]
+        pipe.finish_chat()
+        return answers
+    
+    def generate_with_chat_history(pipe: LLMPipeline, questions):
+        chat_history = ChatHistory()
+        for question in questions:
+            chat_history.append({"role": "user", "content": question})
+            decoded_results = pipe.generate(chat_history, max_new_tokens=50, do_sample=False)
+            chat_history.append({"role": "assistant", "content": decoded_results.texts[0]})
+        return chat_history
+    
     questions = [
         '1+1=',
         'What is the previous answer?',
         'Why is the Sun yellow?',
         'What was my first question?'
     ]
```

Comment:
Seems like the same questions list as in `test_llm_pipeline`, would be great to reuse if possible